tests/validator/validator.cpp

```diff
@@ -1160,6 +1160,36 @@ TEST(Validator, unitAmericanSpellingOfUnitsRemoved)
     };
 
     libcellml::Validator validator;
+    libcellml::ModelPtr model = std::make_shared<libcellml::Model>();
```

Comment:
This is not right (the whole change not just this line). There is already a test for variables in the same component.  I'm surprised that it doesn't compile with an error.

Reply:
I had to change two of the tests just to make it compile?